lib/full-site-editing/page-templates.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Support for page templates.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Load the page templates in Gutenberg.
+ *
+ * @param array    $templates Page templates.
+ * @param WP_Theme $theme     WP_Theme instance.
+ * @param WP_Post  $post      The post being edited, provided for context, or null.
+ * @return array (Maybe) modified page templates array.
+ */
+function gutenberg_load_fse_page_templates( $templates, $theme, $post ) {
```

Comment:
not blocking, but we should start dropping the "fse" term in functions, etc, to something more general like `load_block_page_templates`